integration_tests/integration_tests/tests/test_all_columns_anomalies.py

```diff
@@ -0,0 +1,152 @@
+from datetime import date, timedelta
+from typing import Any, Dict, List
+
+from data_generator import DATE_FORMAT, generate_dates
+from dbt_project import DbtProject
+
+TIMESTAMP_COLUMN = "updated_at"
+DBT_TEST_NAME = "elementary.all_columns_anomalies"
+DBT_TEST_ARGS = {
+    "timestamp_column": TIMESTAMP_COLUMN,
+    "column_anomalies": ["null_count"],
+}
+
+
+def test_anomalyless_all_columns_anomalies(test_id: str, dbt_project: DbtProject):
+    dates = generate_dates(base_date=date.today() - timedelta(1))
+    data: List[Dict[str, Any]] = sum(
```

Comment:
Not sure I understand the requirement of using a `sum()` here (also to the rest).

Reply:
Ohh, I see, personally find [this syntax](https://github.com/elementary-data/dbt-data-reliability/pull/433/files/781fee3d28ff3e9a99dff0198faffa8d3b22fef6#diff-c47f996a703879330b835a9dd776b9c32cbfa5a1184c13fbbed50a8a9e98b46dR33) clearer but not critical.